### PR TITLE
fit(AAiT-web_g3): remove comments and unused codes

### PR DESCRIPTION
**Pull request by dawit-andargachew**
- remove unused codes like comments and console.log
- move static file to data folder
- and some modification on integration all components

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/store/features/auth/index.ts [hunk 1]
```diff
@@ -14,21 +24,93 @@ export const authApi = createApi({
...
 
-export const { useLoginMutation, useRegisterMutation,usePasswordResetMutation } = authApi;
\ No newline at end of file
+export const {
+  useLoginMutation,
+  useRegisterMutation,
+  usePasswordResetMutation,
+} = authApi;
+
+// // Function to get the token from localStorage
```
Comment (Segnicho) [inline aait/web/group-3/starter_project_group3/store/features/auth/index.ts:56]: get rid of these comments

File: aait/web/group-3/starter_project_group3/app/layout.tsx [hunk 2]
```diff
@@ -20,6 +21,7 @@ export default function RootLayout({
         <ReduxProvider>
```
Comment (Segnicho) [inline aait/web/group-3/starter_project_group3/app/layout.tsx:21]: The header and footer should be wrapped with the redux provider.

**Review by Segnicho — CHANGES_REQUESTED** [review]
I have left you some comments

File: aait/web/group-3/starter_project_group3/store/features/auth/index.ts (hunk 1 above)
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/store/features/auth/index.ts:56]: resolved

File: aait/web/group-3/starter_project_group3/app/profile/accountsetting/page.tsx [hunk 3]
```diff
@@ -15,42 +15,47 @@ const AccountSetting = () => {
...
+
     event.preventDefault(); // Prevent default form submission behavior
     passwordReset({
-      oldPassword: confirmPassword,
-      newPassword: currentPassword,
+      oldPassword: currentPassword,
+      newPassword: confirmPassword,
     })
       .unwrap()
       .then((response: any) => {
```
Comment (Segnicho) [inline aait/web/group-3/starter_project_group3/app/profile/accountsetting/page.tsx:30]: get rid of these console logs, and console error

**Review by Segnicho — APPROVED** [review]
LGTM

**Pull request merged**